src/convertToObject.js

```diff
@@ -5,8 +5,16 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  let arr = sourceString.split(';');
```

Comment:
Variable naming like `arr` decreases code readability. Consider using a more descriptive name, such as `styleRules` or `declarations` to reflect the content it holds.